src/shell/layout/tiling/mod.rs

```diff
@@ -3088,6 +3110,7 @@ impl TilingLayout {
                     Some(None),
                     None,
                     None,
+                    theme,
```

Comment:
Couldn't we use `self.theme` here and skip passing the theme to `element_under` (through all of the stack up to `surface_under`)? (That kinda feels wrong to me.)